wherehows-web/tests/integration/components/datasets/containers/dataset-compliance-test.js

```diff
@@ -3,19 +3,10 @@ import { setupRenderingTest } from 'ember-qunit';
 import { render, waitUntil, find } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
 import { initialComplianceObjectFactory } from 'wherehows-web/constants';
-import { startMirage } from 'wherehows-web/initializers/ember-cli-mirage';
 
 module('Integration | Component | datasets/containers/dataset compliance', function(hooks) {
   setupRenderingTest(hooks);
 
-  hooks.beforeEach(function() {
-    this.server = startMirage();
```

Comment:
Same comment as above and for all cases below as well. ps verify if the value is actually returned in the container even if the test passes : - )

Reply:
Yep, it is fine.